LayoutTests/imported/w3c/web-platform-tests/interfaces/invokers.tentative.idl

```diff
@@ -11,5 +11,5 @@ interface InvokeEvent : Event {
 
 dictionary InvokeEventInit : EventInit {
     Element? invoker = null;
-    DOMString action = "auto";
+    DOMString action = "";
```

Comment:
I forgot to update this IDL in https://github.com/WebKit/WebKit/pull/25652, but this is the correct IDL as of the commit in that PR (and is also, at the time of writing, the version in the main branch over on the WPT repo).